src/components/calcite-checkbox/calcite-checkbox.scss

```diff
@@ -1,15 +1,12 @@
 // scale
 :host([scale="s"]) {
   --calcite-checkbox-size: 12px;
-  top: 0.1em;
 }
 :host([scale="m"]) {
   --calcite-checkbox-size: 16px;
-  top: 0.15em;
 }
 :host([scale="l"]) {
   --calcite-checkbox-size: 20px;
-  top: 0.25em;
```

Comment:
Ooh this is nice, this will help on the new `calcite-tile-select` component I'm working on.  Noticed its position being slightly off because of these `top` values.